bgpd/bgp_zebra.c

```diff
@@ -3260,6 +3260,17 @@ static int bgp_zebra_process_srv6_locator_delete(ZAPI_CALLBACK_ARGS)
 				XFREE(MTYPE_BGP_SRV6_SID,
 				      bgp_vrf->vpn_policy[AFI_IP6].tovpn_sid);
 		}
+
+		// refresh per-vrf tovpn_sid
+		tovpn_sid = bgp_vrf->tovpn_sid;
+		if (tovpn_sid) {
+			tmp_prefi.family = AF_INET6;
+			tmp_prefi.prefixlen = 128;
```

Comment:
IPV6_MAX_BITLEN

Reply:
Fixed in https://github.com/FRRouting/frr/pull/11673/commits/059820c65586b51705aabb11b47d59dfa379ea96.